## 57% of end-end tests have passed
```
Total executed tests: 56
Failed tests: 22
Expected to fail tests: 2
Passed tests: 32
```
```
IDs of failed tests: 702807,702786,702850,703629,703495,702732,702948,702809,740221,703202,740220,741054,702808,702958,740222,702957,739307,702845,702777,703297,702851,704615 
```
```
IDs of expected to fail tests: 702844,703503 
```
<h3>Failed tests (22)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/reinstate-mecuryo&PR_ID=21923&APK_NAME=21923.apk&TR_CASE_IDS=702807,702786,702850,703629,703495,702732,702948,702809,740221,703202,740220,741054,702808,702958,740222,702957,739307,702845,702777,703297,702851,704615">Rerun failed tests</a></li><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768719">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Button` by `xpath`: `//*[@text="Edit message"]` </div><div>Device 2: Find `Button` by `xpath`: `//*[@text="Edit message"]` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:823: in test_community_edit_delete_message_when_offline
    self.channel_2.edit_message_in_chat(message_to_edit, message_after_edit)
../views/chat_view.py:655: in edit_message_in_chat
    self.element_by_translation_id("edit-message").double_click()
../views/base_element.py:113: in double_click
    [self.find_element().click() for _ in range(2)]
../views/base_element.py:113: in <listcomp>
    [self.find_element().click() for _ in range(2)]
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 2: Button by xpath: `//*[@text="Edit message"]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESJaEQe05CN3wdXJ01">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYkNCMzro5wI6io68">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestFallbackMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768725">test_fallback_with_correct_seed_phrase</a>, id: 740221 </b></td></tr><tr><td>

```
Test setup failed: critical/test_fallback.py:39: in prepare_devices
    self.home_3.handle_contact_request(self.user_name_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 3: Button by xpath:`//*[contains(@text, 'shpGqKsn4tnYKRmC3Uqp')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768724">test_fallback_sync_with_error</a>, id: 740220 </b></td></tr><tr><td><p><blockquote><div>Device 3: Tap on found: Button </div><div>Device 3: Attempt 0 is successful clicking close-activity-center </div></blockquote></p>

```
Test setup failed: critical/test_fallback.py:39: in prepare_devices
    self.home_3.handle_contact_request(self.user_name_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 3: Button by xpath:`//*[contains(@text, 'shpGqKsn4tnYKRmC3Uqp')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768728">test_fallback_add_key_pair</a>, id: 741054 </b></td></tr><tr><td>

```
Test setup failed: critical/test_fallback.py:39: in prepare_devices
    self.home_3.handle_contact_request(self.user_name_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 3: Button by xpath:`//*[contains(@text, 'shpGqKsn4tnYKRmC3Uqp')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768726">test_fallback_validate_seed_phrase</a>, id: 740222 </b></td></tr><tr><td>

```
Test setup failed: critical/test_fallback.py:39: in prepare_devices
    self.home_3.handle_contact_request(self.user_name_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 3: Button by xpath:`//*[contains(@text, 'shpGqKsn4tnYKRmC3Uqp')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768685">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 2: Attempt 0 is successful clicking close-activity-center </div></blockquote></p>

```
Test setup failed: critical/chats/test_group_chat.py:63: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shVnM4cgNeWzBaupawec')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8zxBdDxiK53qS196">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESPBpbozCLswzJi3Rg">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVL2Qci6vtIm8jFFD">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768714">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:63: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shVnM4cgNeWzBaupawec')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768675">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:63: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shVnM4cgNeWzBaupawec')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768710">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:63: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shVnM4cgNeWzBaupawec')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768686">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:63: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shVnM4cgNeWzBaupawec')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768711">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:63: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shVnM4cgNeWzBaupawec')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768706">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td>

```
Test setup failed: activity_center/test_activity_center.py:383: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'sha6jGHrdAS2YzXtkub7')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768705">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 2: Attempt 0 is successful clicking close-activity-center </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:383: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'sha6jGHrdAS2YzXtkub7')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESDSwLHJ9cvXEPKW7Q">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESSCpNuhCTPrH8eCdR">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768723">test_deep_links_communities</a>, id: 739307 </b></td></tr><tr><td><p><blockquote><div>Device 1: Waiting 35s for dapp to load </div><div>Device 1: Find `Text` by `accessibility id`: `community-title` </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:105: in test_deep_links_communities
    if self.community_view.community_title.text != text:
../views/base_element.py:406: in text
    text = self.find_element().text
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: Text by accessibility id: `community-title` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES7w7hMXvCksSRXj0G">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768697">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `tab-contacts` </div><div>Device 1: Find `Button` by `accessibility id`: `tab-contacts` </div></blockquote></p>

```
activity_center/test_activity_center.py:64: in test_activity_center_contact_request_decline
    self.home_1.contacts_tab.click_until_presence_of_element(self.home_1.pending_contact_request_text)
../views/base_element.py:109: in click_until_presence_of_element
    raise NoSuchElementException("%s element not found" % desired_element.name)
 Text element not found; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2u4GDgxOo6Jcbt7x">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESDbDh7KFS0sEmHSCz">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768680">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 2: Attempt 0 is successful clicking close-activity-center </div></blockquote></p>

```
activity_center/test_activity_center.py:187: in test_add_contact_field_validation
    self.home_2.handle_contact_request(new_username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shk8xDnrDyx81BqL3psH')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2u4GDgxOo6Jcbt7x">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESDbDh7KFS0sEmHSCz">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768698">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `ChatsTab` by `accessibility id`: `chats-stack-tab` </div><div>Device 1: Tap on found: ChatsTab </div></blockquote></p>

```
activity_center/test_activity_center.py:116: in test_activity_center_contact_request_accept_swipe_mark_all_as_read
    self.home_1.notifications_unread_badge.wait_for_visibility_of_element(30)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 1: BaseElement by accessibility id:`activity-center-unread-count` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2u4GDgxOo6Jcbt7x">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESDbDh7KFS0sEmHSCz">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768684">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 2: Attempt 0 is successful clicking close-activity-center </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:852: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shbjyKXcFYbsUA6ip9RB')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES9Y3kfAk3TygnRTlg">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESfwSUs5IZ0jE227lW">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768717">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:852: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shbjyKXcFYbsUA6ip9RB')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Can't navigate to a community channel before joining the community, https://github.com/status-im/status-mobile/issues/21776]]</b><br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768704">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:852: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shbjyKXcFYbsUA6ip9RB')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Can't navigate to a community channel before joining the community, https://github.com/status-im/status-mobile/issues/21776]]</b><br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768687">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:852: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shbjyKXcFYbsUA6ip9RB')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768695">test_community_leave</a>, id: 702845 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:852: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shbjyKXcFYbsUA6ip9RB')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr></tbody></table></details><h3>Expected to fail tests (2)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768694">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'https://m.youtube.com/watch?v=Je7yErjEVt4')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-4']/android.widget.TextView[2]` </div><div>Device 2: Element EmojisNumber text is equal to 1 </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:650: in test_community_links_with_previews_github_youtube_twitter_gif_send_enable
    self.errors.verify_no_errors()
base_test_case.py:176: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 No preview is loaded for url https://youtu.be/Je7yErjEVt4
E    No preview is loaded for url https://www.youtube.com/watch?v=XN-SVmuJH2g&list=PLbrz7IuP1hrgNtYe9g6YHwHO6F3OqNMao
E    No preview is loaded for url https://m.youtube.com/watch?v=Je7yErjEVt4 
```

<b>[[Youtube links preview is not loaded on LambdaTest emulators, needs investigation]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESJaEQe05CN3wdXJ01">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYkNCMzro5wI6io68">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768716">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div># STEP: Check Status community tags in the Discover communities screen</div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:298: in test_community_discovery
    self.errors.verify_no_errors()
base_test_case.py:176: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'Status' is not in the Discover Communities list. 
```

<b>[[Might fail if the Discover curation dapp is too slow to load > 2 mins]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESAQbVhCqWWfhZ39L1">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (32)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768707">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESJaEQe05CN3wdXJ01">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYkNCMzro5wI6io68">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768689">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESJaEQe05CN3wdXJ01">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYkNCMzro5wI6io68">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768700">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESJaEQe05CN3wdXJ01">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYkNCMzro5wI6io68">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768693">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESJaEQe05CN3wdXJ01">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYkNCMzro5wI6io68">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768709">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESJaEQe05CN3wdXJ01">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYkNCMzro5wI6io68">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768690">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESJaEQe05CN3wdXJ01">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYkNCMzro5wI6io68">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768702">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESJaEQe05CN3wdXJ01">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYkNCMzro5wI6io68">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768692">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESJaEQe05CN3wdXJ01">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYkNCMzro5wI6io68">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768691">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESJaEQe05CN3wdXJ01">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYkNCMzro5wI6io68">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768677">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESAQbVhCqWWfhZ39L1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768712">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESAQbVhCqWWfhZ39L1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768708">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESAQbVhCqWWfhZ39L1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768701">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESAQbVhCqWWfhZ39L1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768696">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESAQbVhCqWWfhZ39L1">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768679">test_links_deep_links_profile</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES7w7hMXvCksSRXj0G">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768718">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES7w7hMXvCksSRXj0G">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768727">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768722">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768715">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESGxjxouynEydbmRrQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES4GfawVHDFwz3VGKR">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768682">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESGxjxouynEydbmRrQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES4GfawVHDFwz3VGKR">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768683">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESGxjxouynEydbmRrQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES4GfawVHDFwz3VGKR">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768703">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESBcCcSXHPfBDSGJgQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESXKC1qORqTO09JEGx">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768721">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768720">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768681">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpJG11lXwzxTxTDwQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESl9QEknsk7ngwF782">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768678">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpJG11lXwzxTxTDwQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESl9QEknsk7ngwF782">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768688">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpJG11lXwzxTxTDwQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESl9QEknsk7ngwF782">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768673">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpJG11lXwzxTxTDwQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESl9QEknsk7ngwF782">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768676">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpJG11lXwzxTxTDwQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESl9QEknsk7ngwF782">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768699">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpJG11lXwzxTxTDwQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESl9QEknsk7ngwF782">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768674">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpJG11lXwzxTxTDwQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESl9QEknsk7ngwF782">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3768713">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpJG11lXwzxTxTDwQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESl9QEknsk7ngwF782">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/contexts/wallet/sheets/buy_token/view.cljs
+++ b/src/status_im/contexts/wallet/sheets/buy_token/view.cljs
@@ -7,7 +7,7 @@
             [utils.re-frame :as rf]))
 
 (defn- crypto-on-ramp-item
-  [{:keys [name description fees logo-url site-url recurrent-site-url urls-need-parameters]
+  [{:keys [name fees logo-url site-url recurrent-site-url urls-need-parameters]
     :as   provider}
    _
    _
@@ -22,18 +22,16 @@
                       (rn/open-url (if (= tab :recurrent) recurrent-site-url site-url))))
                   [site-url recurrent-site-url tab])]
     [quo/settings-item
-     {:title             name
-      :description       :text
-      :description-props {:text description}
-      :tag               :context
-      :tag-props         {:icon    :i/fees
-                          :context fees}
-      :action            :arrow
-      :action-props      {:alignment :flex-start
-                          :icon      :i/external}
-      :image             :icon-avatar
-      :image-props       {:icon logo-url}
-      :on-press          open-url}]))
+     {:title        name
+      :tag          :context
+      :tag-props    {:icon    :i/fees
+                     :context fees}
+      :action       :arrow
+      :action-props {:alignment :flex-start
+                     :icon      :i/external}
+      :image        :icon-avatar
+      :image-props  {:icon logo-url}
+      :on-press     open-url}]))
 
 (def ^:private tabs
   [{:id    :one-time

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v9.1.0",
-    "commit-sha1": "4ec885a91f2857a30d1838213000f0a5a91fd52a",
-    "src-sha256": "1fkvhazv7v2vyqpffx3r4p3aii5ai464nx2pdfgn6b6gmb24bxdq"
+    "version": "v9.2.0",
+    "commit-sha1": "41743c6ca0bb0a1ff4510bfa3575948d477e6284",
+    "src-sha256": "1awd04bsxkglbgd0nk5mdz2sm17pxkcj47hfslczsppww7yhmc9n"
 }

